vignettes/planet.Rmd

```diff
@@ -297,12 +305,78 @@ table(results$Predicted_ethnicity)
 Because 'Ambiguous' samples might have different mixtures of ancestries, it 
 might be inadequate to adjust for them as one group in an analysis of admixed
 populations (e.g. 50/50 Asian/African should not be considered the same group 
-as 50/50 Caucasian/African). One solution would be to simply remove these
+as 50/50 European/African). One solution would be to simply remove these
 samples. Another would be to adjust for the raw probabilities-in this case, use
 only two of the three probabilities, since the third will be redundant
 (probabilities sum to 1). If sample numbers are large enough in each group,
 stratifying downstream analyses by ethnicity might also be a valid option.
 
+## Early-onset preeclampsia (EOPE)
+
+To calculate the probability of EOPE of placental DNAm chorionic villi samples, 
+we rely on 45 predictive CpGs. 
+
+In this example, we load the validation data used in the paper and 
+estimate the EOPE probability of the samples. Note that the function must 
+be run on a matrix with the full set of CpG probes in either the 
+450K or 850K arrays - the reason for this is that all 45 predictive CpGs must 
+be present for prediction to be completed. 
+
+It is recommended that data is normalized using BMIQ prior to prediction. 
+
+Note that samples must be rows and CpGs must be columns. The default threshold 
+for classification used to assign labels is 55%; if the users wishes to use 
+other threshold, different labels can be assigned based on the output 
+probabilities. 
+
+```{r include=FALSE}
```

Comment:
for now, we can set these to FALSE, and focus on vignette in a different PR / branch